src/main/java/useCases/createCorporatePage/PageCreationFailed.java

```diff
@@ -0,0 +1,5 @@
+package useCases.createCorporatePage;
+
+public class PageCreationFailed extends RuntimeException{
```

Comment:
You should rename this to PageCreationFailedException and put it in its own package within useCases/createCorporatePage. Having the name include exception makes it easier to understand what the purpose of the class is just from the name, and putting it in its own package keeps the exceptions separate from the regular code classes, especially in case you ever need to add another exception.